oid4vci/demo/frontend/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use an official Node.js runtime as the base image
+FROM node:14
```

Comment:
Node 14 has been past its EOL for a while now, any reason not to update it to a newer version?

Reply:
Good question; I think this may be influenced by the `create-react` template but I think we should be able to just bump this. @burdettadam?

Reply:
Good catch! We should update the image.